### PR TITLE
Add Paper Plugin Installation Guide

### PR DESCRIPTION
**Pull request by sulu5890**
Especially unsure about the `Finding Plugins` section. Do we want to link spigot? I feel like we pretty much have to. I wouldn't be super comfortable including the section at all if it weren't linked, as that would be a pretty significant disservice to people finding plugins.

### CODE REVIEW

File: docs/paper/getting-started/getting-plugins.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+   downloaded ends in `.jar`. Some plugins also distribute as `.zip` files, in which case you will
+   need to extract the file and locate the `.jar` for your platform, often labeled `bukkit` or
+   `paper`.
+2. Once you have the plugin downloaded locally, locate the `plugins` folder from the root directory
+   of your Paper server.
+3. Drag and drop the plugin file (`.jar`) into the `plugins` folder. If you are using a shared
+   hosting service, you may need to use their web panel or SFTP to upload the plugin, however, the
+   procedure will be the same.
+4. Restart your server. The plugin should load.
+5. Check your work. Once the server has finished loading, run the `/plugins` command in-game or type
```
Comment (MiniDigger) [inline docs/paper/getting-started/getting-plugins.md:48]: add something here about that it should be green, if its read, see troubleshooting

File: docs/paper/getting-started/getting-plugins.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+toc_max_heading_level: 4
+---
+
+# Getting Plugins
+
+Plugins are the way to extend the functionality of Paper, beyond the configuration files, and one of
```
Comment (kennytv) [inline docs/paper/getting-started/getting-plugins.md:7]: The inserted "beyond the configuration files" with commas reads a bit weird. I'd probably just remove the commas

File: docs/paper/getting-started/getting-plugins.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+#### Invalid plugin.yml
+
+If you see something closer to this:
+
+```log
+Could not load 'plugins/MyAwesomePlugin-1.0.0.jar' in folder 'plugins'
+org.bukkit.plugin.InvalidDescriptionException: Invalid plugin.yml
+```
+
+This means that what you have download isn't a valid Paper plugin. This is generally caused by one
```
Comment (kennytv) [inline docs/paper/getting-started/getting-plugins.md:88]:
```suggestion
This means that what you have downloaded isn't a valid Paper plugin. This is generally caused by one
```

File: docs/paper/getting-started/getting-plugins.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+## Installing Plugins
+
+1. Once you've found the plugin you'd like to install, download it. Ensure the file you have
+   downloaded ends in `.jar`. Some plugins also distribute as `.zip` files, in which case you will
+   need to extract the file and locate the `.jar` for your platform, often labeled `bukkit` or
+   `paper`.
+2. Once you have the plugin downloaded locally, locate the `plugins` folder from the root directory
+   of your Paper server.
+3. Drag and drop the plugin file (`.jar`) into the `plugins` folder. If you are using a shared
+   hosting service, you may need to use their web panel or SFTP to upload the plugin, however, the
```
Comment (kennytv) [inline docs/paper/getting-started/getting-plugins.md:45]:
```suggestion
   hosting service, you may need to use their web panel or SFTP to upload the plugin; however, the
```

**Comment by MiniDigger** [comment]
looking good, I agree that linking to spigot feels wrong, maybe we can change the order? forums, dbo, spigot? idk
but I agree that not linking to spigot would be worse.
hangar wen?

File: docs/paper/getting-started/getting-plugins.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+#### Something Else
+
+If you see an error, but it isn't similar to one of the above, attempt to read it yourself. While
+the full error may be large and scary, you likely only have to read the first one or two lines to
+understand what is going on. If you're not sure, do not hesitate to reach out for support on our
+[Discord](https://discord.gg/papermc) in the `#paper-help` channel.
+
+### If nothing is logged
+
+If nothing is logged, your server likely not attempting to load any plugins. The conditions needed
```
Comment (kennytv) [inline docs/paper/getting-started/getting-plugins.md:108]:
```suggestion
If nothing is logged, your server is likely not attempting to load any plugins. The conditions needed
```

File: docs/paper/getting-started/getting-plugins.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+for the server to load a plugin are as follows:
+
+1. The file is at the root of the `plugins` folder, relative to its working directory. This is
+   usually the same folder as the server JAR file. **Subdirectories of the `plugins` folder will not
+   be checked.** All plugins must be in the root folder.
+2. The file ends in `.jar`. If your plugin does not end in `.jar`, what you have download may not be
```
Comment (kennytv) [inline docs/paper/getting-started/getting-plugins.md:114]:
```suggestion
2. The file ends in `.jar`. If your plugin does not end in `.jar`, what you have downloaded may not be
```

File: docs/paper/getting-started/getting-plugins.md [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+   a plugin. Note that some plugins distribute multiple jars as `.zip` files. If this is the case,
+   you must extract them before installing the plugin.
```
Comment (kennytv) [inline docs/paper/getting-started/getting-plugins.md:116]:
```suggestion
   you have to extract them before installing the plugin.
```

File: docs/paper/getting-started/getting-plugins.md [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+
+If both of these are true, and you still see no logs, please reach out for support on our
```
Comment (kennytv) [inline docs/paper/getting-started/getting-plugins.md:118]:
```suggestion
If both of these are true and you still see no logs, please reach out for support on our
```

**Comment by kennytv** [comment]
I personally think it's fine to link to Spigot, that's just what people will be linked to or find the most when searching for plugins... at least until Hangar is a thing

**Comment by jacksyyy** [comment]
Duplicate of #37. no steal

**Review by Machine-Maker — APPROVED** [review]

**Pull request merged**